src/app/form-email/form-email.component.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-unused-expressions */
+import {Component, OnInit, forwardRef} from '@angular/core';
+import {
+  ControlValueAccessor,
+  Validators,
+  FormGroup,
+  FormControl,
+  NG_VALUE_ACCESSOR,
+} from '@angular/forms';
+
+@Component({
+  selector: 'app-form-email',
+  templateUrl: 'form-email.component.html',
+  styleUrls: ['form-email.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => FormEmailComponent),
+      multi: true,
+    },
+  ],
+})
+export class FormEmailComponent implements OnInit, ControlValueAccessor {
+  public emailForm: FormGroup = new FormGroup({
+    // eslint-disable-next-line @typescript-eslint/unbound-method
+    email: new FormControl('', [Validators.required, Validators.email]),
+  });
+
+  // constructor() {}
+
+  ngOnInit(): void {}
+
+  public onTouched: () => void = () => {};
+  writeValue(val: any): void {
```

Comment:
MAI usare any esplicitamente

Reply:
queste funzioni le abbiamo praticamente copiate da internet, per risolvere il problema dei componenti inclusi in altri componenti. Poter riutilizzare il codice è utilissimo e sono il primo a promuoverlo, ma si sta rivelando complicato